### PR TITLE
Add teca link to atmosphere MED

### PR DESCRIPTION
**Pull request by rbeucher**
# ACCESS Community Hive 

Thank you for submitting a pull request to the ACCESS Hive Project. 

## Description

Add link to The Toolkit for Extreme Climate Analysis

## Type of change

Please delete options that are not relevant.

- [X] New link / content

## Checklist:

- [x] The new content is accessible and located in the appropriate section.
- [x] My changes do not break navigation and do not generate new warnings
- [x] I have checked that the links are valid and point to the intended content.
- [x] I have checked my code/text and corrected any misspellings

Please tag the **reviewers team** in a comment below by prepending an `@` in front of `ACCESS-Hive/reviewers` when ready.

### CODE REVIEW

File: docs/model_evaluation/atmosphere.md [hunk 1]
```diff
@@ -40,6 +40,15 @@ Links to the code repository and documentation for each METplus component are pr
 The PMP is used to provide “quick-look” objective comparisons of Earth System Models (ESMs) with one another and available observations. Results are produced in the context of all model simulations contributed to CMIP6 and earlier CMIP phases. Currently, the comparisons emphasize metrics of large- to global-scale annual cycle and both tropical and extra-tropical modes of variability. Ongoing work in v1.x development branches include established statistics for ENSO, MJO, regional monsoons, and high frequency characteristics of simulated precipitation.
 
 
+## [Toolkit for Extremes Climate Analysis (TECA)][teca-web]  {{ recom }}
+
+[**Documentation**][teca-doc]
+[**Tutorials**][teca-tutorials]
+[**Sources**][teca-source]
```
Comment (ccarouge) [inline docs/model_evaluation/atmosphere.md:47]:
```suggestion
[**Documentation**][teca-doc] |
[**Tutorials**][teca-tutorials] |
[**Sources**][teca-source]
```

File: docs/model_evaluation/atmosphere.md [hunk 2]
```diff
@@ -40,6 +40,15 @@ Links to the code repository and documentation for each METplus component are pr
 The PMP is used to provide “quick-look” objective comparisons of Earth System Models (ESMs) with one another and available observations. Results are produced in the context of all model simulations contributed to CMIP6 and earlier CMIP phases. Currently, the comparisons emphasize metrics of large- to global-scale annual cycle and both tropical and extra-tropical modes of variability. Ongoing work in v1.x development branches include established statistics for ENSO, MJO, regional monsoons, and high frequency characteristics of simulated precipitation.
 
 
+## [Toolkit for Extremes Climate Analysis (TECA)][teca-web]  {{ recom }}
```
Comment (ccarouge) [inline docs/model_evaluation/atmosphere.md:43]: Why is it recommended? Do we have experience with it?

**Review by ccarouge — COMMENTED** [review]
I want to clarify the tagging before it goes in.

File: docs/model_evaluation/atmosphere.md (hunk 2 above)
Comment (rbeucher) [reply docs/model_evaluation/atmosphere.md:43]: Good point. Changed to community

**Review by ccarouge — APPROVED** [review]
All good.

**Pull request merged**